package.json

```diff
@@ -5,6 +5,9 @@
   "files": [
     "lib/**/*"
   ],
+  "bin": {
```

Comment:
Its this easy to tell npm that you offer a CLI?

Reply:
pretty much